containers/withAdminContainer.ts

```diff
@@ -0,0 +1,27 @@
+import { Context } from '../@types/helpers'
+import { isAdmin } from '../helpers/isAdmin'
+import { withUserContainer } from './withUserContainer'
+import _ from 'lodash'
```

Comment:
Ah just noticed this -- i think we arent using anything from lodash in this file anymore, so it can be cleaned up

Reply:
ill make a new PR that fixes this.